packages/documentation/src/content/docs/overview/concepts/accounting.mdx

```diff
@@ -1,139 +1,278 @@
 
-## Liquidity
+:::note
+If you and your peer transact in the same asset (there's no currency conversion) and you both provide your customers only with wallet addresses denominated in that asset, then there will be no movement into/from the corresponding asset liquidity account.
+
+For example, you and your peer transact in USD and only provide your customers with USD wallet addresses. One of your customers sends $10 to your peer's customer. There's no movement from your USD asset liquidity account because there was no currency conversion. There is, however, an [outgoing payment liquidity account](/overview/concepts/accounting#outgoing-payment-liquidity-accounts) created to support the transaction.
+:::
 
-Rafiki tracks liquidity using the Interledger Protocol, which is a clearing protocol, without physically holding funds.
+You can add a liquidity threshold for each asset liquidity account via the [`updateAsset`](https://rafiki.dev/apis/graphql/backend/mutations/#updateasset) mutation's `liquidityThreshold` input argument.
 
-### Asset liquidity
+When a threshold is entered, the [`asset.liquidity_low`](/integration/requirements/webhook-events#asset-liquidity-low) webhook event will notify you if an asset account's liquidity drops below the threshold.
 
-Asset liquidity represents the value, denominated in a given asset, that Rafiki has at its disposal in which to send or forward ILP packets. Asset liquidity increases when packets are received and decreases when packets are sent. Asset liquidity is always positive and cannot fall below zero.
+You should define and adjust asset liquidity based on your liquidity risk profile. You can deposit or withdraw asset liquidity as needed through [Rafiki Admin](/admin/admin-user-guide#edit-asset) or by using the [Backend Admin API](/admin/manage-liquidity#asset-liquidity).
 
-Additionally, you must provide asset liquidity if you are exchanging one currency for another. Asset liquidity ensures there is enough value available to handle transactions in the specified assets.
+<details>
+<summary>Asset liquidity example - cross-currency transactions</summary>
+Your Rafiki instance is configured for two assets: EUR and USD.
 
-You should define and adjust asset liquidity based on your liquidity risk profile.
+- Rafiki holds an asset liquidity account for both EUR and USD.
+- You’ve set the asset scale of both currencies to 0.
+- Your starting EUR liquidity is 10 and your USD liquidity is 50.
 
-#### Asset liquidity examples
+**Cross-currency transaction #1:**
 
-Your Rafiki instance is configured with two assets: EUR and USD. You've set both to have an asset scale of 0. Your EUR liquidity is 10, and your USD liquidity is 50.
+<Steps>
+  1. Rafiki receives packets from a peer. These packets are all denominated in
```

Comment:
To be more specific, it is:
1. 10 EUR move between the peer liquidity account to the asset liquidity account
2. Exchange rate applied, 12 USD moves from asset liquidity account to incoming payment liquidity account

Reply:
The second example, if we don't have 12 USD in the USD asset account, the transfer will fail